src/web/js/trove/chart-lib.js

```diff
@@ -1207,36 +1152,66 @@
       legends.push(legend);
       data.addColumn('number', legend);
       data.addColumn({type: 'string', role: 'tooltip', 'p': {'html': true}});
+      data.addColumn({id: 'i0', type: 'number', role: 'interval'});
+      data.addColumn({id: 'i1', type: 'number', role: 'interval'});
     });
 
+    // console.log('legends is', legends);
+    // console.log('data I is', data);
+
     combined.forEach((p, i) => {
-      /*
-      x | n n n | y | n n n n n n n n n n n n
-            i         combined.length - i - 1
-      */
```

Comment:
This comment was informative about the shape of the data – did the shape change/is the comment irrelevant? Was it wrong/should it be documented somewhere what the new shape is?